framework.go

```diff
@@ -175,6 +175,32 @@ func (f *Framework) ApplyCluster(ctx context.Context, cluster *application.Clust
 		return nil, err
 	}
 
+	ns := cluster.GetNamespace()
+	cm_name := fmt.Sprintf("%s-app-operator-user-values", cluster.Name)
+
+	// Create a ConfigMap with the user values for app-operator
+	configMapData := make(map[string]string, 0)
+	values := `
+	service.operatorkit.resyncPeriod: 1m
+	`
+	configMapData["values"] = values
+
+	cm := corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cm_name,
+			Namespace: ns,
+		},
+		Data: configMapData,
+	}
+
+	if err := f.mcClient.CreateOrUpdate(ctx, &cm); err != nil {
+		return nil, fmt.Errorf("failed to create app-operator user values: %v", err)
+	}
```

Comment:
In this instance I think I'd opt to just log the error and continue as we can still work without this configmap being there.

Reply:
Changed